addons/web/static/tests/core/hotkeys/hotkey_service_tests.js

```diff
@@ -78,6 +78,38 @@ QUnit.test("data-hotkey", async (assert) => {
     comp.destroy();
 });
 
+QUnit.test("invisible data-hotkey", async (assert) => {
+    assert.expect(1);
+
+    class MyComponent extends Component {
+        onClick() {
+            assert.step("click");
+        }
+    }
+    MyComponent.template = xml`
+    <div>
+      <button t-on-click="onClick" data-hotkey="b" style="display: none;"/>
+    </div>
+  `;
+
+    const key = "alt+b";
+    triggerHotkey(key);
```

Comment:
Verify that it does work, maybe ?
It's just that the fix commit being placed before the imp commit that removes the "altIsOptional" option, I have a little doubt... I did not check.
If it works, you don´t need to change that though.